### ISSUE TITLE
Text flees to the right side of the screen when I use C-x C-+ to zoom

### ISSUE THREAD

**Issue by DustyQuince** [syl20bnr/spacemacs#12973]
My text goes to the right side of the window after I use C-x C-+ to zoom in on Spacemacs. 

## Context

This occurs in every mode I have tried and it does not occur when I start emacs using emacs -q. The text starts with a small indent, but every motion I make with my cursor after the initial C-x C-+ results in all text shifting right until it ends with text crowded on the right side of the window.

## Environment

OS: Ubuntu 18.04, Linux Kernel 4.15.0-70
GUI: GNOME 3.28.2
Emacs: 26.3
Spacemacs: 0.200.13

## Process

1. User edits normally upon start-up
2. User presses C-x C-+, + + to adjust font size
3. User moves cursor within buffer and text flees to right side of window

## Expected result

I expect the text to get bigger but not move within the buffer

## Current result

When I move the cursor using movement commands, the text goes to the right side of the screen. If I switch buffers and switch back, it sometimes is in the normal place, but adjusts itself to the right side of the screen again upon cursor movement. This does not occur when using emacs -q

[not obligatory, but suggest fixes or reasons for the bug]

## `right side text` screenshot
![Screenshot from 2019-11-22 09-49-11](https://user-images.githubusercontent.com/49379463/69439094-3f41b800-0d14-11ea-889a-cf99b7486877.png)

----

**Comment by rnwolf** [comment]
Checked to see what happens on the **develop** branch of spacemacs on both widows and ubuntu. 

C-x C-+, + + or - - to adjusts font size as expected.

**Comment by DustyQuince** [comment]
Ok. I downloaded the develop branch and initially it worked as expected. When I used relative line numbers, which was my default on my own setup, the problem happened again. It also happend when I used absolute line numbers, and it happened whether line numbers were default or toggled on, and stopped when they were toggled off. If you turn line numbers on and adjust size, does it still work as expected?

**Comment by rnwolf** [comment]
I have just turned on spacemacs relative line numbers with `SPC t r`.
Then `C-x C-+` after which I can use + or - or 0 to zoom in out and reset. Will keep it on and see if any funny business occurs later.

**Comment by duianto** [comment]
This is a duplicate of this (and other similar issues):
Linum-mode creates margins on zoomed-in font #9382

Emacs 26 introduced built in line numbers support.
The Spacemacs develop branch uses the built in once if the Emacs version is 26+.
Use native line numbers in Emacs 26 #10675

Your system info shows that your using 26.3, so it should be working.

**Comment by DustyQuince** [comment]
It seems I had linum-mode 'relative turning on in my .spacemacs file, so I guess it overrode the default to use the Emacs 26 native? Anyway, the display-line-numbers doesn't make it quite as easy to default to relative line numbers, so I added a hook to apply that. I don't know if that is the most efficient solution, but it works. These are the lines I added to my .spacemacs user-config section:
  (when (version<= "26.0.50" emacs-version )
    (global-display-line-numbers-mode))
  (setq display-line-numbers 'relative)
  (add-hook 'find-file-hook
            '(lambda () (setq display-line-numbers 'relative)))
